roles/kubernetes-apps/ingress_controller/cert_manager/defaults/main.yml

```diff
@@ -7,6 +7,12 @@ cert_manager_nodeselector: {}
 cert_manager_dns_policy: "ClusterFirst"
 cert_manager_dns_config: {}
 
+## Allow http_proxy, https_proxy and no_proxy environment variables
+## Issue https://github.com/kubernetes-sigs/kubespray/issues/2933
```

Comment:
Could you remove this line? IMO linking to the issue doesn't add much value here...

Reply:
Understood. I will remove this line